code/game/gamemodes/hivemind/objectives.dm

```diff
@@ -68,7 +68,7 @@
 
 /datum/objective/hivemind/awaken/update_explanation_text()
 	if(target && target.current)
-		explanation_text = "Turn [target.name], the [!target_role_type ? target.assigned_role : target.special_role], into an awakened vessel."
+		explanation_text = "Turn [target.name], the [!target_role_type ? target.get_station_role(TRUE) : target.get_special_role(TRUE)], into an awakened vessel."
```

Comment:
`target_role_type ` checks it. it will only tell station role at most, but if an objective var `target_role_type` is set true, then we'll get special name.